office365.json

```diff
@@ -10,11 +10,11 @@
     "product_version_regex": ".*",
     "publisher": "Splunk",
     "license": "Copyright (c) 2017-2022 Splunk Inc.",
-    "app_version": "2.4.9",
-    "utctime_updated": "2022-01-19T19:06:27.000000Z",
+    "app_version": "2.4.14",
```

Comment:
Did an exta version bump happen at some point? I see 2.4.13 in most places, but the version is `2.4.14` here

Reply:
I see it gets bumped at https://github.com/splunk-soar-connectors/msgoffice365/commit/15596b739b8ddbb80374fc1e7a6b078e5c002c75

Reply:
The readme and release notes show different version from what's in the JSON

Reply:
I think what happened is that new commits were added to the next branch *after* the `Start Release` action was run.

Reply:
I haven't tried it yet, but I think re-running `Start Release` will cause it to re-use the existing PR.